plugins/usi-dock/fu-usi-dock-mcu-device.c

```diff
@@ -712,9 +712,9 @@ fu_usi_dock_mcu_device_insert_cb(gpointer user_data)
 	fwupd_request_set_kind(request, FWUPD_REQUEST_KIND_IMMEDIATE);
 	fwupd_request_set_id(request, FWUPD_REQUEST_ID_INSERT_USB_CABLE);
 	fwupd_request_add_flag(request, FWUPD_REQUEST_FLAG_ALLOW_GENERIC_MESSAGE);
-	fwupd_request_set_message(
-	    request,
-	    "The update will continue when the device USB cable has been re-inserted.");
+	fwupd_request_set_message(request,
+				  "Please insert the USB cable, then, new firmware will take "
+				  "effect after dock LED blinking finish.");
```

Comment:
finish**es**, no?

Reply:
Yes, after the LED stops flashing, the upgrade will be completed and take effect. Then the host can be able to detect the insertion.

Reply:
Yeah I was just making a suggestion in your wording.

Reply:
Thanks, these statements are directly defined and approved by the customer.

Reply:
But the text doesn't make sense.

Reply:
What about we have `Please insert the USB cable to allow the update to continue.`